### ISSUE TITLE
Add support for Laravel 10

### ISSUE THREAD

**Issue by szabrzyski** [protonemedia/laravel-ffmpeg#474]
Composer error:
- pbmedia/laravel-ffmpeg[8.2.0, ..., 8.2.2] require illuminate/contracts ^9.0 -> satisfiable by illuminate/contracts[v9.0.0, ..., v9.52.0].

**Comment by pascalbaljet** [comment]
Available in v8.3.0 🔥

**Comment by CharlieBrownCharacter** [comment]
@pascalbaljet First of all, great package. It really solves a lot of issues for a lot of people. 

I'm trying to install it in a laravel 10 project with `composer require pbmedia/laravel-ffmpeg:8.3.0` but I get the following error:

```
./composer.json has been updated
Running composer update pbmedia/laravel-ffmpeg
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires pbmedia/laravel-ffmpeg 8.3.0 -> satisfiable by pbmedia/laravel-ffmpeg[8.3.0].
    - pbmedia/laravel-ffmpeg 8.3.0 requires illuminate/contracts ^9.0|^10.0 -> found illuminate/contracts[v9.0.0-beta.1, ..., 9.x-dev, v10.0.0, ..., 10.x-dev] but these were not loaded, likely because it conflicts with another require.


Installation failed, reverting ./composer.json and ./composer.lock to their original content.
```

I tried searching for ` illuminate/contracts` in my `composer.json` but wasn't able to find it. Im guessing its a Laravel dependency. Would you mind helping me out on this one?

Here's my `composer.json`:

```json
{
    "name": "laravel/laravel",
    "type": "project",
    "description": "The Laravel Framework.",
    "keywords": ["framework", "laravel"],
    "license": "MIT",
    "require": {
        "php": "^8.2",
        "aws/aws-sdk-php": "~3.0",
        "guzzlehttp/guzzle": "^7.7",
        "laravel/framework": "^11.0",
        "laravel/sanctum": "dev-develop",
        "laravel/tinker": "dev-develop",
        "league/flysystem-aws-s3-v3": "^3.0",
        "php-ffmpeg/php-ffmpeg": "*"
    },
    "require-dev": {
        "fakerphp/faker": "^1.9.1",
        "laravel/pint": "^1.0",
        "laravel/sail": "dev-develop",
        "mockery/mockery": "^1.4.4",
        "nunomaduro/collision": "^7.0",
        "phpunit/phpunit": "^10.0"
    },
    "autoload": {
        "psr-4": {
            "App\\": "app/",
            "Database\\Factories\\": "database/factories/",
            "Database\\Seeders\\": "database/seeders/"
        }
    },
    "autoload-dev": {
        "psr-4": {
            "Tests\\": "tests/"
        }
    },
    "scripts": {
        "post-autoload-dump": [
            "Illuminate\\Foundation\\ComposerScripts::postAutoloadDump",
            "@php artisan package:discover --ansi"
        ],
        "post-update-cmd": [
            "@php artisan vendor:publish --tag=laravel-assets --ansi --force"
        ],
        "post-root-package-install": [
            "@php -r \"file_exists('.env') || copy('.env.example', '.env');\""
        ],
        "post-create-project-cmd": [
            "@php artisan key:generate --ansi"
        ]
    },
    "extra": {
        "branch-alias": {
            "dev-master": "11.x-dev"
        },
        "laravel": {
            "dont-discover": []
        }
    },
    "config": {
        "optimize-autoloader": true,
        "preferred-install": "dist",
        "sort-packages": true,
        "allow-plugins": {
            "pestphp/pest-plugin": true,
            "php-http/discovery": true
        }
    },
    "minimum-stability": "dev",
    "prefer-stable": true
}

```
